README.md

```diff
@@ -76,6 +81,13 @@ the variable pizza inside your method.
 Your Answer:
 ```js
 // write code here
+var pizza = function () {
+  temperature: 70,
+  bake: function (setTemperature) {
```

Comment:
In this case, you don't need to pass in an argument into the bake method per say. It's definitely not wrong, but you would need to pass in 300 each time your calling that method